components/sup/src/manager/mod.rs

```diff
@@ -613,6 +629,19 @@ impl Suitability for SuitabilityLookup {
     }
 }
 
+fn obtain_process_lock(fs_cfg: &FsCfg) -> Result<()> {
+    match OpenOptions::new().write(true).create_new(true).open(&fs_cfg.proc_lock_file) {
+        Ok(_) => Ok(()),
+        Err(_) => Err(sup_error!(Error::NoProcessLock)),
```

Comment:
Right - even if you don't put the logic in to release the lock, writing something in it thats actionable is smart.